src/ray/gcs/subscription_executor.cc

```diff
@@ -48,12 +48,11 @@ Status SubscriptionExecutor<ID, Data, Table>::AsyncSubscribe(
 
   auto on_subscribe = [this](RedisGcsClient *client, const ID &id,
                              const std::vector<Data> &result) {
+    RAY_LOG(DEBUG) << "Subscribe received update of id " << id;
```

Comment:
```suggestion
    RAY_LOG(DEBUG) << "Received update of id " << id;
```
Also, It'd be nice to also print the type of `Data` here, otherwise this log is not very useful. 
Can you try if this works https://stackoverflow.com/a/28738540?